pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2012,6 +2012,10 @@ type AzblobStorageProvider struct {
 	// SecretName is the name of secret which stores the
 	// azblob service account credentials.
 	SecretName string `json:"secretName,omitempty"`
+	// StorageAccount is the storage account of the azure blob storage
+	StorageAccount string `json:"storageAccount,omitempty"`
```

Comment:
How about adding some comments for `If Backup.Spec.StorageProvider.Azblob.StorageAccount is specified, then use this to set backup-manager pod env. Otherwise get storage account from secret as before`